config/license_finder.yml

```diff
@@ -0,0 +1,6 @@
+---
+project_name: Porta
```

Comment:
Where did you see this? In README I only see `license_finder project_name add Porta` which alters `dependency_decisions.yml`, not the config file

Reply:
It is used as the page title when generating a report in HTML format.

Reply:
It doesn't work for me locally. Whatever I put there, project name is always `porta`.